elk/tests/utils_generation/test_gen_parser.py

```diff
@@ -0,0 +1,15 @@
+from elk.utils_generation.parser import get_parser
+
+
+def test_args_no_lowercase():
```

Comment:
Ditto here

Reply:
Ahhh, overlooked that. Fixed.

Reply:
Great, will merge as soon as you push it